js/hang-ui/src/Components/watch/WatchUIContextProvider.tsx

```diff
@@ -66,6 +78,14 @@ export default function WatchUIContextProvider(props: WatchUIContextProviderProp
 		}
 	};
 
+	const setActiveRenditionValue = (name: string | undefined) => {
+		const hangWatchEl = props.hangWatch();
+
+		if (hangWatchEl) {
```

Comment:
These sorts of conditions worry me, because it means we can silently skip signal updates.

@jdreetz I think this happens a lot in the code base. I think we should gate the entire UI behind `HangWatch` so it's not optional. I don't think every UI element should handle what to do if the core element doesn't exist.

Reply:
@kixelated I agree. I'll make a separate PR for that unless you think we should do it in this PR.

Reply:
Mine should be ready soon, but looks like this is already merged so I'll refactor to support it.